webviewer/src/components/Auth/Form/AuthForm.styles.js

```diff
@@ -1,17 +1,12 @@
 import styled, { css } from 'styled-components';
-import { Link } from 'react-router-dom';
-
-export const StyledLink = styled(Link)`
-  ${({ theme }) => css`
-    color: ${theme.colors.primary.p2};
-    font-weight: ${theme.fontWeights[1]};
-    text-decoration: none;
-  `}
-`;
 
 export const StyledMessage = styled.span`
   ${({ theme }) => css`
     color: ${theme.colors.black};
     opacity: 0.7;
   `}
 `;
+
+export const LinkWrap = styled.div`
```

Comment:
maybe change LinkWrap to something like ForgorPassLinkWrap? 

because it is used not as common style